signer/src/testing/storage/postgres.rs

```diff
@@ -38,4 +39,29 @@ impl PgStore {
         .await
         .map_err(Error::SqlxQuery)
     }
+
+    /// Get all key-rotation events that have been logged in the database for the
+    /// given aggregate key.
+    pub async fn get_key_rotation_events_for_aggregate_key(
+        &self,
+        aggregate_key: &PublicKey,
+    ) -> Result<Vec<model::KeyRotationEvent>, Error> {
+        sqlx::query_as::<_, model::KeyRotationEvent>(
+            r#"
+            SELECT
+                txid
+              , block_hash
+              , address
+              , aggregate_key
+              , signer_set
+              , signatures_required
+            FROM sbtc_signer.rotate_keys_transactions
+            WHERE aggregate_key = $1
+            "#,
+        )
+        .bind(aggregate_key)
+        .fetch_all(self.pool())
+        .await
+        .map_err(Error::SqlxQuery)
+    }
```

Comment:
nit: this doesn't seem to be used here nor in the other PRs, can we remove it? unless it was supposed to be used in `wait_for_key_rotation_event`

Reply:
I've got some unpublished code using it. Can remove it here and co-locate it with the using code.